packit_service/worker/parser.py

```diff
@@ -383,8 +385,10 @@ def parse_issue_comment_event(event) -> Optional[IssueCommentEvent]:
         issue_id = nested_get(event, "issue", "number")
         action = event.get("action")
         comment = nested_get(event, "comment", "body")
-        if action != "created" or not issue_id or not comment:
+        comment_object = event.get("comment")
+        if action != "created" or not (issue_id and comment and comment_object):
             return None
+        comment_object = GithubIssueComment(comment_object)
```

Comment:
If you look in ogr, these constructors expect the comment object from the dependency (e.g. PyGithub or python-gitlab). Here you are given webhook = `comment_object` is some kind of a JSON, not the thing `GithubIssueComment` processes.

Reply:
Also thinking about it, it is likely that this kind of attributes (e.g. projects, pull request / issue or comments) are initialized via property once they are needed.